admindb/sqlite/auth_fallback.go

```diff
@@ -39,23 +39,23 @@ func (ah AuthFallback) Check(name, password string) (interface{}, error) {
 	return found.ID, nil
 }
 
-func (ah AuthFallback) Create(ctx context.Context, name string, password []byte) error {
+func (ah AuthFallback) Create(ctx context.Context, name string, password []byte) (int64, error) {
```

Comment:
I wonder if we can have a type for the invite, instead of obliquely passing around `int64`–or is that a bit much?

Reply:
Right. Technically this leaks (or requires) that all the admindbs have numerical ID types.

I honestly don't think it makes much sense to put another level of indirection here. (The way out would be to have interfaces but what would the method on that be but returning an integer again or do a bunch of assertions to get the actual value back from underlying implementations)

We could have `type ID int64` in admindb to make it more concrete what these numbers are (could also be affected rows or something entirely different) they will need to be cast like `int64(userID)` in more cases then one would think. Go is quite picky there.

Reply:
> We could have type ID int64 in admindb to make it more concrete what these numbers are (could also be affected rows or something entirely different) they will need to be cast like int64(userID) in more cases then one would think. Go is quite picky there.

Yeah that was basically the only thing I was advocating for, just to make the interfaces a bit more readable & self-documenting, but if you think it would cause a lot of casts then I don't think it is worth it here.

*resolved*

Reply:
I think it should be fine until we end up with another class/group of integers we have to work with and so far I don't see that happening in this project.